app/src/main/admin/components/ModellingGroups/List/ModellingGroupsListContent.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import {compose} from "recompose";
+import { connect } from 'react-redux';
+import {clone} from "lodash";
+
+import { ModellingGroup } from "../../../../shared/models/Generated";
+import { ModellingGroupListItem } from "./ModellingGroupListItem";
+
+import {AdminAppState} from "../../../reducers/adminAppReducers";
+
+interface ModellingGroupsProps {
+    groups: ModellingGroup[]
+}
+
+export const ModellingGroupsListContentComponent: React.SFC<ModellingGroupsProps> = (props: ModellingGroupsProps) => {
+    if (Array.isArray(props.groups) && props.groups.length) {
```

Comment:
Don't these two conjuncts check for the same thing?

Reply:
These are different. first one checks data type, second one the length of it.
[] - is array , but length is 0

Reply:
But this just checks whether the length exists, not whether it's equal to 0. Was the latter the intention?

Reply:
no, it's not correct, it not checks if length exists if checks value, so if it is 0, condition will not pass

Reply:
Oh, javascript falsey magic :laughing: